### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by AbraaoAlves** [dcTeam23/face2face#3] [Ideas]
Precisamos de ideias de logo.

Se você tem alguma ideia de arte, logo para usarmos no projeto essa é a hora!

**Comment by AbraaoAlves** [comment]
Sugestão 1 - fazer um logo simétrico parecido com esse

![image](https://github.com/dcTeam23/face2face/assets/608731/7f5fd7b3-90ad-47c3-a07f-736cba943d06)


> podemos discutir outras cores

**Reply by jmtannus** [reply]
Boa idéia!
Certo professor, vou pensar aqui. :) 
Rodrigo já deu umas ideias legais, vou me inspirar nelas também. :)

**Reply by jmtannus** [reply]
Fiz a criação dessa logo, de acordo com a que o senhor nos enviou. Usei a fonte Magneto. As cores foram inspiradas na pele de dois rostos diferentes. 
![Logo_face2face_Jmt_color](https://github.com/dcTeam23/face2face/assets/61756665/cab3dc3e-1c12-42bb-a4ff-97d340d63f9c)

**Reply by jmtannus** [reply]
Segue abaixo a logo com fundo transparente
![Logo_face2face](https://github.com/dcTeam23/face2face/assets/61756665/f52386ea-a938-4d52-9686-ef22c50dc2ac)

**Reply by jmtannus** [reply]
Caso não goste ou queira mudar algo, pode ser super sincero e falar. Isso se estende a todo mundo, blz turma?! 
Um forte abraço!

**Reply by jmtannus** [reply]
Olá turma!
Segue essa logo pequena, com fundo transparente:
![F2F_1](https://github.com/dcTeam23/face2face/assets/61756665/31592ebd-fd3a-423e-a169-dfd2e84c99f8)

**Reply by jmtannus** [reply]
e esse é menos deitado:
![F2F_1_1](https://github.com/dcTeam23/face2face/assets/61756665/0861e54a-efa3-4940-9e51-777da72f77e3)

**Comment by RodrigoNet0** [comment]
![image](https://github.com/dcTeam23/face2face/assets/134714036/4ba367d1-2e79-41c3-8f6a-7770c2b537de)

![image](https://github.com/dcTeam23/face2face/assets/134714036/d942b53f-69cc-48c5-b540-19afaa6d0657)




Sugestões criada pela IA bing

**Reply by jmtannus** [reply]
Eita! Ficou show! Gostei mais do primeiro, que não identificamos o gênero.

**Reply by jmtannus** [reply]
e o nome do segundo tá melhor. :)
